bcda/main_test.go

```diff
@@ -335,11 +411,24 @@ func setupJobArchiveFile(s *MainTestSuite, email string, modified time.Time, acc
 }
 
 func (s *MainTestSuite) TestCleanArchive() {
+
+	// init
 	autoMigrate()
-	os.Setenv("FHIR_ARCHIVE_DIR", "../bcdaworker/data/test/archive")
 	const Threshold = 30
 	now := time.Now()
 
+	// condition: FHIR_ARCHIVE_DIR doesn't exist
+	os.Unsetenv("FHIR_ARCHIVE_DIR")
```

Comment:
Learning so much from this PR. All this time I've been setting env vars to empty strings because I didn't know this function existed.